src/Controller/Admin/BoardCrudController.php

```diff
@@ -55,5 +56,7 @@ public function configureFields(string $pageName): iterable
             ->hideOnForm()
         ;
         yield IntegerField::new('defaultDeadline', 'Default Deadline(days)');
+
+        yield TextareaField::new('statuses', 'Statuses');
```

Comment:
Should probably be translated via 'templates/translations/admin.html.twig'

Reply:
Fixed in 7398264